lib/errbit_gitlab_plugin/issue_tracker.rb

```diff
@@ -125,13 +125,24 @@ def create_issue(title, body, reported_by = nil)
     private
 
     #
-    # Tries to find a project with the given name in the given Gitlab installation
-    # and returns its ID (if any)
+    # Replaces `/` with the corresponding url encoding
     #
-    def gitlab_project_id(gitlab_url = options[:endpoint], token = options[:api_token], project = options[:path_with_namespace])
-      @project_id ||= with_gitlab(gitlab_url, token) do |g|
-        g.projects.auto_paginate.detect { |p| p.path_with_namespace == project }.try(:id)
+    def urlized_project_name(project_name = options[:path_with_namespace])
+      project_name.gsub('/', '%2F')
```

Comment:
I'm not sure if its appropriate here, but you can use URI#encode if the intention is to encode the whole string. The second argument to encode is an additional set of characters to encode beyond the standard ones. `URI.encode(project_name, '/')` for example.